<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AVTOR</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>10</y>
      <width>661</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QGroupBox" name="groupBox_checkIn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>391</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Check In</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>201</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_CheckIn_login"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_CheckIn_password_1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_CheckIn_password_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>81</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_CheckIn_login">
          <property name="text">
           <string>login:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_CheckIn_password_1">
          <property name="text">
           <string>password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_CheckIn_password_2">
          <property name="text">
           <string>password:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>180</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_CheckIn_Cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_CheckIn_SighUp">
          <property name="text">
           <string>Sigh up</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_sighUp_img">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>20</y>
        <width>311</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_sighIn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>391</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Sigh In</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>201</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit_sighIn_login"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_sighIn_password"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>81</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_sighIn_login">
          <property name="text">
           <string>login:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_sighIn_password">
          <property name="text">
           <string>password:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>180</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_sighIn_SighUp">
          <property name="text">
           <string>Sigh up</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_sighIn_SighIn">
          <property name="text">
           <string>Sigh in</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_sighIn_img">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>20</y>
        <width>311</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_main_SighOut">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>13</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sigh out</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_main_yourName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>13</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Your username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_main_login">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>13</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>628</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_new_copyright">
       <attribute name="title">
        <string>Add New Copyright</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_add_path">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>90</y>
          <width>291</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_add_path">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>120</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Add path to file...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_add_go">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>200</y>
          <width>191</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Register for the copyright</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_your_copyrights">
       <attribute name="title">
        <string>Your Registered Products</string>
       </attribute>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>321</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string>Selected copyright:</string>
        </property>
        <widget class="QLabel" name="label_main_dateAndTime_notImportant">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>211</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Date and time of registration:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_main_dateAndTime_important">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>63</y>
           <width>251</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>92</y>
           <width>301</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_main_hash1_notImportant">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>First file hash:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_main_hash1a_important">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>281</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>&quot;00000000000000000000000000000000-</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_main_hash1b_important">
         <property name="geometry">
          <rect>
           <x>26</x>
           <y>146</y>
           <width>281</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>00000000000000000000000000000000&quot;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_main_hash2_notImportant">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>190</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Second file hash:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_main_hash2b_important">
         <property name="geometry">
          <rect>
           <x>26</x>
           <y>226</y>
           <width>281</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>00000000000000000000000000000000&quot;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_main_hash2a_important">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>281</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>&quot;00000000000000000000000000000000-</string>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>301</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_test_product">
       <attribute name="title">
        <string>Test New Product</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_test_path">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>120</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Add path to file...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_test_go">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>200</y>
          <width>151</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Test this profuct</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_test_path">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>90</y>
          <width>291</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_blockchain_info">
       <attribute name="title">
        <string>Blockchain Info</string>
       </attribute>
       <widget class="QLabel" name="label_info_numberBlocks_notImportant">
        <property name="geometry">
         <rect>
          <x>165</x>
          <y>40</y>
          <width>271</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>The number of blocks in the blockchain:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_info_numberBlocks_important">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>70</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_info_numberCopyrights_notImportant">
        <property name="geometry">
         <rect>
          <x>152</x>
          <y>120</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>The number of copyrights in the blockchain:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_info_numberCopyrights_important">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>150</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_info_numberUsers_notImportant">
        <property name="geometry">
         <rect>
          <x>158</x>
          <y>210</y>
          <width>291</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>The number of users registered in AVTOR:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_info_numberUsers_important">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>240</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>126</x>
          <y>90</y>
          <width>351</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>148</x>
          <y>180</y>
          <width>311</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_main_img">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>0</y>
        <width>140</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
